Library/Formula/freexl.rb

```diff
@@ -27,4 +27,8 @@ def install
       doc.install "html"
     end
   end
+
+  test do
+    assert File.exist? "#{lib}/libfreexl.a"
```

Comment:
No problem, so we would also have to fix some more packages that have this kind of tests.

Sent from my iPhone

> On 25 Jul 2015, at 13:30, Xu Cheng <notifications@github.com> wrote:
> 
> In Library/Formula/freexl.rb:
> 
> > @@ -27,4 +27,8 @@ def install
> >        doc.install "html"
> >      end
> >    end
> > +
> > +  test do
> > +    assert File.exist? "#{lib}/libfreexl.a"
> Can you write a simple C program? You can see tinyxml2 for example.
> 
> —
> Reply to this email directly or view it on GitHub.
>